frontend/src/components/pages/acls/Models.ts

```diff
@@ -29,7 +29,7 @@ export type AclFlat = {
 
 export type AclPrincipalGroup = {
     principalType: PrincipalType;
-    principalName: string;
+    principalName: string | '*';
```

Comment:
```suggestion
    // This can only ever be a literal, or match anything (star in that case). No prefix or postfix matching
    principalName: string | '*';
```